cmd/commands/login.go

```diff
@@ -63,7 +63,7 @@ func tokenAuth(ctx context.Context) {
 	}
 }
 
-func deviceAuth(ctx context.Context) {
+func DeviceAuth(ctx context.Context) {
```

Comment:
I kinda want to allow any type of automatic auth here - including passing in a token - but it feels odd to duplicate the flags.

It just supports `DeviceAuth` right now for local dev; we should probably look at allowing `--token` and `--username` at a later date.